barretenberg/cpp/src/barretenberg/ultra_honk/decider_verifier.cpp

```diff
@@ -55,14 +55,21 @@ template <typename Flavor> bool DeciderVerifier_<Flavor>::verify()
         return false;
     }
 
-    const auto opening_claim = Shplemini::compute_batch_opening_claim(accumulator->verification_key->circuit_size,
-                                                                      commitments.get_unshifted(),
-                                                                      commitments.get_to_be_shifted(),
-                                                                      claimed_evaluations.get_unshifted(),
-                                                                      claimed_evaluations.get_shifted(),
-                                                                      multivariate_challenge,
-                                                                      Commitment::one(),
-                                                                      transcript);
+    auto opening_claim = Shplemini::compute_batch_opening_claim(accumulator->verification_key->circuit_size,
+                                                                commitments.get_unshifted(),
+                                                                commitments.get_to_be_shifted(),
+                                                                claimed_evaluations.get_unshifted(),
+                                                                claimed_evaluations.get_shifted(),
+                                                                multivariate_challenge,
+                                                                Commitment::one(),
+                                                                transcript);
+    Shplemini::remove_shifted_commitments(opening_claim,
+                                          Flavor::TO_BE_SHIFTED_WITNESSES_START,
```

Comment:
These indexes should be passed as a single data structure constructed in the flavor

Reply:
done, created a struct for this